### ISSUE TITLE
[Blog] Review blog post draft - Context Propagation

### ISSUE THREAD

**Issue by lauracowen** [microprofile/microprofile-marketing#169]
In draft on Wordpress: https://microprofile.io/wp-admin/post.php?post=6370&action=edit

**Comment by aeiras** [comment]
Hey Laura, I will own the review :)

**Comment by aeiras** [comment]
@lauracowen 
Blog header missing, crospostcrosspost looks ok. 
<img width="404" alt="image" src="https://user-images.githubusercontent.com/10731224/63544475-2b4ea600-c4da-11e9-8c5f-67c34eafffb8.png"> 

Please update and comment via this medium to enable publishing content :)

**Comment by aeiras** [comment]
since you are already a MP content editor, you can help publish Nathan Rauh's content, however his blog needs to have his name added as the creator and writer of the blog. 
Currently the blog has you as the author which must be adjusted to Nathan Rauh. Nathan is already a Contributor of WP, which makes it easy to adjust. FYI. 
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/10731224/63544792-ccd5f780-c4da-11e9-93fd-762825b1ed68.png">

**Comment by lauracowen** [comment]
@aeiras

Page header- I'm not sure I understand the page header image thing. Isn't the image displayed only when the post is hosted by MicroProfile.io?

Nathan - ah ok, I didn't check whether he already had an account - that makes it easier - I wasn't comfortable with it being in my name! Now fixed.

**Comment by aeiras** [comment]
You are amazing @lauracowen :)

Blog is [published](https://microprofile.io/2019/08/23/microprofile-context-propagation-open-liberty/), please note that white space where the title is placed. There, should you & author desire it, a pic can be possible. Not a requirement. 
I see this is Nathan's 2nd post, awesome. He still doesn't have a pic, which only him can adjust via wordpress contributor's access. I don't have his git ID. If you could reach him, that would be a +1 action. 

I took the liberty to publish the blog, I adjusted the introductory hyperlink in the description of the blog that welcomes the reader to the original blog. 

**In the future** when you can start review and publish content without extra eyes :)  If needed pulled me, or others' listed in the WordPress [volunteer sheet](https://docs.google.com/document/d/1q0Xw4XJcoVq1V5vPWe9KR__WKoGc0e-X5cD3tnBsIKw/edit) anyone in to help. 

Thank you, hope to see you in a few weeks at OC1- SFO!

**Comment by lauracowen** [comment]
> You are amazing @lauracowen :)
> 
> Blog is [published](https://microprofile.io/2019/08/23/microprofile-context-propagation-open-liberty/), please note that white space where the title is placed. There, should you & author desire it, a pic can be possible. Not a requirement.

@aeiras Thank you for your help! I just noticed that the URL is showing as part of the abstract on the website: 
![image](https://user-images.githubusercontent.com/3322532/63879104-e25b8d80-c9c2-11e9-8026-40bd8d25585c.png)
And the redirect doesn't work. I will fix it but I'm not sure how the redirect is set for blog posts hosted elsewhere?

> I see this is Nathan's 2nd post, awesome. He still doesn't have a pic, which only him can adjust via wordpress contributor's access. I don't have his git ID. If you could reach him, that would be a +1 action.
I've asked Nathan to add a pic.

> **In the future** when you can start review and publish content without extra eyes :) If needed pulled me, or others' listed in the WordPress [volunteer sheet](https://docs.google.com/document/d/1q0Xw4XJcoVq1V5vPWe9KR__WKoGc0e-X5cD3tnBsIKw/edit) anyone in to help.
The instructions say that two editors are required so I wasn't sure whether I could just publish it. Or are two editors required only when the blog post is hosted on MP.io and that needs to be reviewed as well as published?

> Thank you, hope to see you in a few weeks at OC1- SFO!
Thank you! Hopefully meet you in person at some point, though maybe not OC1.

**Comment by aeiras** [comment]
Laura, 

By doing- brake stuff, only then the process is likely to improve 💪🏼, don’t hold back. 
As an editor, you cannot brake the website, so everything is easily fixable! Do it! 💜

**Comment by lauracowen** [comment]
> Laura,
> 
> By doing- brake stuff, only then the process is likely to improve 💪🏼, don’t hold back.
> As an editor, you cannot brake the website, so everything is easily fixable! Do it! 💜

Okay, I can do that but I can't work out how to add a redirect. For example, in [this previous post](https://microprofile.io/wp-admin/post.php?post=6009&action=edit), where is it configured to redirect to the post that is hosted on OpenLiberty.io/blog? The Permalink shows a MP.io domain (which, when loaded, redirects to the openliberty.io domain). I'm confused! :)

**Comment by aeiras** [comment]
What about you

1- delete post
2- ask the author who is already a contributor to crosspost using wiki

note: bc I am one of the MicroProfilers behind the creation of the guidelines, I need to step aside & use your feedback to make the steps easier to own. Your feedback  is awesome. We need a short video on how to crosspost content. 
I will create a new git issue & ask for volunteers to own such video💗💪🏼🧸  thank you 😊

**Comment by lauracowen** [comment]
I've taken the post offline. As I did the original submission of this anyway, I've taken another look at the wiki instructions (which I think have been updated?) and tried adding the link setting in the right sidebar.

But that's not how this previous post seems to be configured: https://microprofile.io/wp-admin/post.php?post=6009&action=edit (when I configure the new post https://microprofile.io/wp-admin/post.php?post=6370&action=edit using the 'link' setting in the right sidebar, the presentation of the post is different with no intro text).
